src/algorithm/translate.rs

```diff
@@ -0,0 +1,196 @@
+use num_traits::{Float, FromPrimitive};
+use types::{Point, Polygon, LineString, Line, MultiPoint, MultiPolygon, MultiLineString};
+
+// translate a slice of points by the given offsets
+fn translation_matrix<T>(x_offset: T, y_offset: T, points: &[Point<T>]) -> Vec<Point<T>>
+    where T: Float
+{
+    points
+        .iter()
+        .map(|point| {
+                 let new_x = point.x() + x_offset;
+                 let new_y = point.y() + y_offset;
+                 Point::new(new_x, new_y)
```

Comment:
Maybe `map_coords`? That's what python-geojson calls it https://pypi.python.org/pypi/geojson#map-coords